### PR TITLE
Used requirejs

### PR DESCRIPTION
**Pull request by zefciu**

### CODE REVIEW

File: ajax_select/static/js/ajax_select.js [hunk 1]
```diff
@@ -320,7 +314,7 @@ var BobAjaxSelect = (function (window, undefined) {
                 var parentDeck = $('#' + $(input).data('parentId') + '_on_deck');
                 var childDeck = $('#' + childId + '_on_deck');
                 // get the original AJAX source from options
-                var options = $(input).data('autocomplete').options;
+                var options = $(input).data('uiAutocomplete').options;
```
Comment (ar4s) [inline ajax_select/static/js/ajax_select.js]: why did you changed this value?

**Comment by xliiv** [comment]
Mixed formatting changes with changes of functionality which is a bad idea in general.

File: ajax_select/static/js/ajax_select.js (hunk 1 above)
Comment (zefciu) [inline ajax_select/static/js/ajax_select.js]: The jquery-ui api change

**Comment by xliiv** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**